src/tasks.md

```diff
@@ -0,0 +1,263 @@
+    pull           Pulls a Kubewarden policy from a given URI
+    push           Pushes a Kubewarden policy to an OCI registry
+    rm             Removes a Kubewarden policy from the store
+    run            Runs a Kubewarden policy from a given URI
+    verify         Verifies a Kubewarden policy from a given URI using Sigstore
+```
+
+Here are a few examples of the commands you should run, depending on the task you want to perform:
+
+- *List the policies*: lists all the policies stored in the local `kwctl` registry
+
+  - Command: `kwctl policies`
+
+- *Obtain the policy*: download and store the policy inside the local `kwctl` registry
+
+  - Command: `kwctl pull <policy URI>`
+  - Example:
+
+  ```shell
+  $ kwctl pull registry://ghcr.io/kubewarden/policies/pod-privileged:v0.1.9
+  
+  $ kwctl policies
+  +--------------------------------------------------------------+----------+---------------+--------------+----------+
+  | Policy                                                       | Mutating | Context aware | SHA-256      | Size     |
+  +--------------------------------------------------------------+----------+---------------+--------------+----------+
+  | registry://ghcr.io/kubewarden/policies/pod-privileged:v0.1.9 | no       | no            | 59e34f482b40 | 21.86 kB |
+  +--------------------------------------------------------------+----------+---------------+--------------+----------+
+  ```
+
+- *Understand how the policy works*: inspect the policy metadata
+
+  - Command: `kwctl inspect <policy URI>`
+  - Example:
+
+  ```shell
+    $ kwctl inspect registry://ghcr.io/kubewarden/policies/pod-privileged:v0.1.9
+    Details
+    title:              pod-privileged
+    description:        Limit the ability to create privileged containers
+    author:             Flavio Castelli
+    url:                https://github.com/kubewarden/pod-privileged-policy
+    source:             https://github.com/kubewarden/pod-privileged-policy
+    license:            Apache-2.0
+    mutating:           false
+    context aware:      false
+    execution mode:     kubewarden-wapc
+    protocol version:   1
+    
+    Annotations
+    io.kubewarden.kwctl 0.1.9
+    
+    Rules
+    ────────────────────
+    ---
+    - apiGroups:
+        - ""
+      apiVersions:
+        - v1
+      resources:
+        - pods
+      operations:
+        - CREATE
+    ────────────────────
+    
+    Usage
+    This policy doesn't have a configuration. Once enforced, it will reject
+    the creation of Pods that have at least a privileged container defined.
+  ```
+
+- *Evaluate the policy*: Assess the policy and, if available, find the right configuration values to match your requirements.
+
+  > NOTE: Familiarity with [Kubernetes REST APIs](https://kubernetes.io/docs/reference/) is a prerequisite.
+
+  - Command: `kwctl run -r <"Kubernetes Admission request" file path> -s <"JSON document" file path> <policy URI>`
+
+    - Command breakdown:
```

Comment:
I'm wary of listing command options in an specific manner, I always find they drift out of sync.

I would prefer to focus on the examples, and then have 1 page on the documentation that lists the kwctl commands in a `man` style. This normally can be automated, in kwctl's case it would be https://rust-cli.github.io/book/in-depth/docs.html.

Reply:
I suggest removing the command breakdown for now. In the future, we could substitutethe output of `kwctl --help` for a link to the docs page that contains the "manpage"-style  docs of kwctl.

Reply:
Removed the command breakdown. And yes, as discussed, I suggest we have a `kwctl` oriented page.